SPECS/python-gast/python-gast.spec

```
@@ -0,0 +1,112 @@
+%global _description %{expand:
+A generic AST to represent Python2 and Python3's Abstract Syntax Tree (AST).
+GAST provides a compatibility layer between the AST of various Python versions,
+as produced by ast.parse from the standard ast module.}
+
+Summary:        Python AST that abstracts the underlying Python version
+Name:           python-gast
+Version:        0.5.3
+Release:        5%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/serge-sans-paille/gast
+Source0:        %{url}/archive/%{version}/gast-%{version}.tar.gz
+BuildRequires:  pyproject-rpm-macros
+BuildRequires:  python3-devel
+BuildRequires:  python3-packaging
+BuildRequires:  python3-pip
+BuildRequires:  python3-wheel
+BuildArch:      noarch
+
+%description %{_description}
+
+%package -n     python3-gast
+Summary:        %{summary}
+
+%description -n python3-gast %{_description}
+
+%prep
+%autosetup -p1 -n gast-%{version}
+
+%generate_buildrequires
+%pyproject_buildrequires -t
+
+
+%build
+%pyproject_wheel
+
+
+%install
+%pyproject_install
+%pyproject_save_files gast
+
+
+%check
+pip3 install tox tox-current-env pytest
```

This is just something we've noticed recently: some tests are unstable and fail for newer version of `pytest`. Could you check what version was used for the successful run of the tests here (it'll be mentioned in the logs right after the `pip3 install` command is run) and "freeze" the dependency to that version? For example:

```suggestion
pip3 install tox tox-current-env pytest==[version]
```